index.html

```diff
@@ -24,20 +24,42 @@
                 
                 <!-- Webpage Title -->
                 <div class="title">
-                    <a>ASTRONOMY DIVE</a>
+                    <a href="./index.html">ASTRONOMY DIVE</a>
                 </div>
     
                 <!-- Navigation Bar -->
                 <nav class="nav-bar">
-                    <ul>
```

Comment:
Removing the lists inside the nav bar made it a lot easier to work with the nav-items. I realized that I made it more complex for myself by nesting the items within lists.